charts/galoy/templates/api-ingress.yaml

```diff
@@ -13,6 +13,9 @@ metadata:
   annotations:
     cert-manager.io/cluster-issuer: {{ .Values.galoy.api.ingress.clusterIssuer }}
 
+    nginx.ingress.kubernetes.io/proxy-read-timeout: "21600" # 6 hours
```

Comment:
is this not missing the unit?

https://docs.nginx.com/nginx-ingress-controller/configuration/ingress-resources/advanced-configuration-with-annotations/

if default is 60s, not sure we really that big of a value? maybe we could try 10 min and see how that works. I think eventually those value should only apply to the websocket, not the public api for normal http request

Reply:
let's try 1h then? 

it should also be seen in regards to our usage.  I'm doubting much sessions last 1h+ in our case. 

longer session make it prone to some attack (like [slowloris](https://www.cloudflare.com/learning/ddos/ddos-attack-tools/slowloris/)) so better to not be overly large here